python/src/ignition/gazebo/server_config.cc

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Let's follow Ignition's conventions for file names, i.e.

`python/src/ignition/gazebo/ServerConfig.cc`

And so on